src/main/java/emu/grasscutter/command/commands/TalentCommand.java

```diff
@@ -87,6 +87,30 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
                 };
                 setTalentLevel(sender, avatar, skillId, newLevel);
             }
+            case "all" -> {
+                if (args.size() < 2) {
+                    sendUsageMessage(sender);
+                    return;
+                }
+                try {
+                    newLevel = Integer.parseInt(args.get(1));
+                } catch (NumberFormatException ignored) {
+                    CommandHandler.sendTranslatedMessage(sender, "commands.talent.invalid_level");
+                    return;
+                }
+                // This stops setTalentLevel from outputting 3 "levels out of range" messages
+                if (newLevel < 1 || newLevel > 15) {
+                    CommandHandler.sendTranslatedMessage(sender, "commands.talent.out_of_range");
+                    return;
+                }
+                // This is small so array is not needed imo
+                skillId = skillDepot.getSkills().get(0);
+                setTalentLevel(sender, avatar, skillId, newLevel);
+                skillId = skillDepot.getSkills().get(1);
+                setTalentLevel(sender, avatar, skillId, newLevel);
+                skillId = skillDepot.getEnergySkill();
+                setTalentLevel(sender, avatar, skillId, newLevel);
```

Comment:
general advice:
```suggestion
                setTalentLevel(sender, avatar, skillDepot.getSkills().get(0), newLevel);
                setTalentLevel(sender, avatar, skillDepot.getSkills().get(1), newLevel);
                setTalentLevel(sender, avatar, skillDepot.getEnergySkill(), newLevel);
```

but specifically for this:
```suggestion
                skillDepot.getSkillsAndEnergySkill().forEach(id -> setTalentLevel(sender, avatar, id, newLevel));
```

Reply:
oh whoops, `getSkillsAndEnergySkill` also hits alternate sprints

Reply:
thanks for the suggestion!